CHANGELOG.md

```diff
@@ -50,6 +51,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - ACMG classification not showing for some causatives
 - Fix bug which caused IGV.js to use hg19 reference files for hg38 data
 - Bug when multiple bam files sources with non-null values are available
+- Cancer variant view has cleaner table data entries for "N/A" data
```

Comment:
I think this should belong to version 4.12.1, since we merged today

Reply:
Ah, sorry - please move it where you want it!